pkg/apis/workspaces/v1alpha2/components.go

```diff
@@ -54,6 +55,14 @@ type ComponentUnion struct {
 	// +optional
 	Container *ContainerComponent `json:"container,omitempty"`
 
+	// Allows importing into the devworkspace docker-compose files
+	// defined in a given manifest. For example this allows the reuse of previously
+	// docker-compose files used to define configuration for managing
+	// multiple containers at the same time.
```

Comment:
Trying to address the points raised (rightfully) by @amisevsk 

```suggestion
	// Allows referencing an existing Compose files so that its abstractions as 
	// Services and Volumes will be included in the resulting development 
	// environment.
```